### PR TITLE
Migrated a couple of samples

### PR DESCRIPTION
**Pull request by bettinaheim**

### CODE REVIEW

File: Samples/algorithms/oracle-synthesis/Program.qs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿namespace IntegerFactorization {
```
Comment (cgranade) [inline Samples/algorithms/oracle-synthesis/Program.qs:1]: I'd suggest a copyright header here, as well as changing the namespace to Microsoft.Quantum.Samples.OracleSynthesis.

File: Samples/algorithms/oracle-synthesis/Program.qs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    open Microsoft.Quantum.Samples.OracleSynthesis;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation Program () : Unit {
```
Comment (cgranade) [inline Samples/algorithms/oracle-synthesis/Program.qs:8]:
```suggestion
    operation RunProgram() : Unit {  
```

File: Samples/characterization/phase-estimation/Program.qs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿namespace PhaseEstimationSample {
```
Comment (cgranade) [inline Samples/characterization/phase-estimation/Program.qs:1]: Similarly here, I'd suggest a copyright header and using the same namespace.

File: Samples/characterization/phase-estimation/Program.qs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    open Microsoft.Quantum.Samples.PhaseEstimation;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation Program () : Unit {
```
Comment (cgranade) [inline Samples/characterization/phase-estimation/Program.qs:8]:
```suggestion
    operation RunProgram() : Unit {
```

File: samples/algorithms/simple-grover/SimpleGrover.qs [hunk 5]
```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Quantum.Samples.SimpleGrover {
     /// # Summary
```
Comment (cgranade) [inline samples/algorithms/simple-grover/SimpleGrover.qs:10]: As discussed offline, I agree with deleting Host.cs. That said, it looks like Host.cs is being used on this docs page, such that we should adapt the Grover quickstart in coordination with doing so: https://github.com/MicrosoftDocs/quantum-docs-pr/blob/master/articles/quickstarts/search.md.

+@KittyYeungQ for awareness and coordination.

File: Samples/algorithms/oracle-synthesis/Program.qs (hunk 1 above)
Comment (bettinaheim) [reply Samples/algorithms/oracle-synthesis/Program.qs]: ups, thanks!

File: Samples/algorithms/oracle-synthesis/Program.qs (hunk 2 above)
Comment (bettinaheim) [reply Samples/algorithms/oracle-synthesis/Program.qs:11]: done

File: Samples/characterization/phase-estimation/Program.qs (hunk 3 above)
Comment (bettinaheim) [reply Samples/characterization/phase-estimation/Program.qs]: done

File: Samples/characterization/phase-estimation/Program.qs (hunk 4 above)
Comment (bettinaheim) [reply Samples/characterization/phase-estimation/Program.qs:11]: done

File: samples/algorithms/simple-grover/SimpleGrover.qs (hunk 5 above)
Comment (bettinaheim) [reply samples/algorithms/simple-grover/SimpleGrover.qs:10]: Ah, sorry. I will leave the change in then, and leave it up to you to coordinate?

**Comment by bettinaheim** [comment]
I need to pick the newer package version for this - I'll add that shortly, which should fix the build.

File: samples/algorithms/simple-grover/SimpleGrover.qs (hunk 5 above)
Comment (cgranade) [reply samples/algorithms/simple-grover/SimpleGrover.qs:10]: That might work in the meantime, yes. I definitely agree with removing Host.cs, just want to make sure that the docs are not impacted as we do so. Thanks for the help!

Comment (bettinaheim) [reply samples/algorithms/simple-grover/SimpleGrover.qs:10]: @cgranade Do you want to set up a suitable feature branch to retarget this to?

Comment (cgranade) [reply samples/algorithms/simple-grover/SimpleGrover.qs:10]: Sounds good, I've gone on and done that, then. Thanks!

**Comment by bettinaheim** [comment]
> Looks good, thank you! As discussed, I've retargeted to feature/standalone so that @KittyYeungQ can test changes in the docs against your changes here. Thank you!

Cool, thanks! I assume you will take care of the merge to master?

**Pull request merged**